File: src/main/java/com/finfamily/FinFamilyApplication.java
package com.finfamily;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
import org.springframework.boot.builder.SpringApplicationBuilder;
import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;


@SpringBootApplication(exclude = {JacksonAutoConfiguration.class})
public class FinFamilyApplication extends SpringBootServletInitializer{

    @Override
    protected SpringApplicationBuilder configure(SpringApplicationBuilder application) {
        return application.sources(FinFamilyApplication.class);
    }

    public static void main(String[] args) {
        SpringApplication.run(FinFamilyApplication.class, args);
    }

}

File: src/main/kotlin/com/bandtec/finfamily/finfamily/FinfamilyApplication.kt
package com.bandtec.finfamily.finfamily;

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = [JacksonAutoConfiguration::class])
class FinfamilyApplication

fun main(args: Array<String>) {
    runApplication<FinfamilyApplication>(*args)
}
